### PR TITLE
Spelling: app, sync, licensed, system settings

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/qml/pages/InfoPage.qml [hunk 1]
```diff
@@ -62,7 +62,7 @@ Page {
             width: parent.width
             horizontalAlignment: Text.AlignHCenter
             wrapMode: Text.WrapAtWordBoundaryOrAnywhere
-            text: i18n.tr("This application let you synchronize with AsteroidOS smartwatches")
+            text: i18n.tr("This app lets you synch with AsteroidOS smartwatches")
```
Comment (jrtberlin) [inline app/qml/pages/InfoPage.qml]: Any particular reason you used 'synch' instead of 'sync'? In your other commit you [are using 'sync'](https://github.com/AsteroidOS/telescope/pull/50/commits/936fea9e308d633cc5f88fab9a92589826cec297#diff-f30a6f85d7ee55488f1b3dd39197c118R103)
I'm not a native speaker so I might be wrong. The spelling just stood out to me (although that's technically a correct but less common spelling)

File: app/qml/pages/MainMenuPage.qml [hunk 2]
```diff
@@ -100,7 +100,7 @@ Page {
             ListItem {
                 enabled: root.watch.timeServiceReady
                 ListItemLayout {
-                    title.text: i18n.tr("Synchronize time")
+                    title.text: i18n.tr("Sync time")
```
Comment (MagneFire) [inline app/qml/pages/MainMenuPage.qml]: Is there any particular reason to change this? I think it is immediately clear what is meant by Synchronize. Sync seems like an unnecessary change.

File: app/qml/pages/InfoPage.qml [hunk 3]
```diff
@@ -71,7 +71,7 @@ Page {
             horizontalAlignment: Text.AlignHCenter
             wrapMode: Text.WrapAtWordBoundaryOrAnywhere
             //TRANSLATORS: Please make sure the URLs are correct
-            text: i18n.tr("Released under the terms of the <a href=\"https://github.com/AsteroidOS/telescope/blob/master/LICENSE\">GNU GPL v3</a>")
+            text: i18n.tr("Copylefted libre software licensed <a href=\"https://github.com/AsteroidOS/telescope/blob/master/LICENSE\">GPLv3+</a>")
```
Comment (MagneFire) [inline app/qml/pages/InfoPage.qml]: I like this :+1:

File: app/qml/pages/WatchesPage.qml [hunk 4]
```diff
@@ -84,7 +84,7 @@ Page {
         anchors.centerIn: parent   
         color: Suru.foregroundColor
         font.pointSize: units.gu(2)     
-        text: i18n.tr("No smartwatches configured yet. Please connect your smartwatch using System Settings.")
+        text: i18n.tr("No smartwatches configured yet. Please connect your smartwatch using the system settings.")
```
Comment (MagneFire) [inline app/qml/pages/WatchesPage.qml]: Using `System Settings` here makes sense as the location to look for is in the app `System Settings`. Any reason to lowercase this here?

File: app/qml/pages/InfoPage.qml (hunk 1 above)
Comment (MagneFire) [reply app/qml/pages/InfoPage.qml]: I agree with what @jrtberlin said here. The best string here would be in my opinion: `This application lets you synchronize with AsteroidOS smartwatches`

File: app/qml/pages/InfoPage.qml (hunk 3 above)
Comment (jrtberlin) [reply app/qml/pages/InfoPage.qml]: I agree with the general concept because it is more descriptive. I would suggest a comma behind "software".  So instead of "`... software licensed GPLv3`" I would write "`... software, licensed GPLv3`".

File: app/qml/pages/WatchesPage.qml (hunk 4 above)
Comment (comradekingu) [reply app/qml/pages/WatchesPage.qml]: If it does what it says on the tin, what relevance does it bare that it is an app (with a proper name)?
To me it sounds like some kind of TV-shop concept. "Open the SmartFridgeDoorTM on your SmartFridge2012.", only now, it is The Door door.

Edit: To clarify, it grinds my goat when concepts that have become basic and accepted, all of a sudden have to be fancified (to make up a word). "Enter a phone number using The Dialer" and so on.

File: app/qml/pages/InfoPage.qml (hunk 1 above)
Comment (comradekingu) [reply app/qml/pages/InfoPage.qml]: @jrtberlin Slip on my end. Good catch. Main point was to avoid "let" in error.

That being said…
@MagneFire The tally is 7 app/s to 2 application/s, this PR getting rid of both of the latter two.
https://hosted.weblate.org/languages/en/asteroidos/search/?offset=1&q=app&sort_by=-priority%2Cposition&checksum=

https://hosted.weblate.org/languages/en/asteroidos/search/?offset=1&q=sync&sort_by=-priority%2Cposition&checksum=    8-2 in favour of "sync".

File: app/qml/pages/MainMenuPage.qml (hunk 2 above)
Comment (comradekingu) [reply app/qml/pages/MainMenuPage.qml]: Just for consistency and brevity. It could be misunderstood as a noun instead of a transitive verb.
Possible middleground "**Sync the time**", which one could possible argue should be "**Sync up the time**". which further should be "**Sync clock**" (which avoids the issues up until this point). If one wants to be even more precise from there "**Sync device clock**". ← I unsurprisingly like this one.

File: app/qml/pages/InfoPage.qml (hunk 1 above)
Comment (MagneFire) [reply app/qml/pages/InfoPage.qml]: @comradekingu Thanks for your feedback. I agree that this stuff should be consistent. That being said. This is a description for the app. The usage of `app` is fine, but notice that on most sites it is referred to synchronization apps:
https://asteroidos.org/wiki/synchronization-clients/ (No mention of `sync`)
https://f-droid.org/en/packages/org.asteroidos.sync/
https://github.com/AsteroidOS/starfish

Also The app for Android is called `AsteroidOS Sync` this makes the tally actually: 4-3 for `sync`. I guess your change is fine because we have something similar in the Android client: `This app lets you sync your Android phone with your AsteroidOS watch`

File: app/qml/pages/WatchesPage.qml (hunk 4 above)
Comment (MagneFire) [reply app/qml/pages/WatchesPage.qml]: Your edit actually makes your point perfectly clear :)
This makes complete sense now :+1:

Comment (comradekingu) [reply app/qml/pages/WatchesPage.qml]: @MagneFire Well we can't have that ;)

Comment (comradekingu) [inline app/qml/pages/WatchesPage.qml]:
```suggestion
        text: i18n.tr("Connect a smartwatch using the system settings first.")
```

File: app/qml/pages/MainMenuPage.qml (hunk 2 above)
Comment (comradekingu) [inline app/qml/pages/MainMenuPage.qml]:
```suggestion
                    title.text: i18n.tr("Sync device clock")
```

**Comment by FlorentRevest** [comment]
Cool, thank you comradekingu!
